### PR TITLE
[WIP]: Added various images for the dark and light theme for the Layer5 website also removed unwanted code within the SVG Files.

### LINKED ISSUE #5467: PNG masquerading as an SVG
### Current Behavior
Under the "Kickstart with Ready-to-Use Templates" section of the https://layer5.io/cloud-native-management/generate-aws-architecture-diagram page, the associated image is is not actually an SVG, but should be.

### Expected Behavior
Re-export image as a true SVG.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Ajay-singh1**
**Description**

This is a `Work in progress` PR.

This PR fixes #5467 

**Notes for Reviewers**

List of websites where this changes took effect.

1.https://layer5.io/cloud-native-management/catalog (3-D Spinning)
2.https://layer5.io/cloud-native-management/kanvas (3-D Spinning)
3.https://layer5.io/cloud-native-management/generate-aws-architecture-diagram
4.https://layer5.io/cloud-native-management/generate-gcp-architecture-diagram
5.https://layer5.io/cloud-native-management/generate-kubernetes-architecture-diagram
7.https://https://layer5.io/solutions/developer-defined-infrastructure
8.https://layer5.io/solutions/cloud-native-deployments-by-diagram

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 997abdb08379b91136b4263ddcef54304d20b221 at: https://6758565d60435f6227f0395a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cb06ee8fff73210a84457b3bbd4e3b05e0dc7337 at: https://675862cf3aba0800813f5053--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bf31c1c0d53e86047dfc868c1818431c8744e328 at: https://675a4807577d84cf83b2cb1c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a223d5734f73f677251fe906aebbb0100d46f0b1 at: https://675a4cc74b7dd2c9047354f6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1d54ea2f7dee8a9825c4b2f2b3c1709198a38eb9 at: https://675a55026eb9b79af7a67bbf--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 94c84b3940b8f2912a2ef06668d48a0d8d3febd0 at: https://675a5c2e4b7dd2dcdf735178--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit dbe16d76b494d5bb227576195ac7ac41cbd36ce5 at: https://675a691eb7f803f2d960ab50--layer5.netlify.app

**Comment by Ajay-singh1** [comment]
@leecalcote Indeed their was a PNG File within this SVG I have fixed it.

**Comment by Ajay-singh1** [comment]
> 🚀 Preview for commit [dbe16d7](https://github.com/layer5io/layer5/commit/dbe16d76b494d5bb227576195ac7ac41cbd36ce5) at: https://675a691eb7f803f2d960ab50--layer5.netlify.app

Final Result.

**Comment by l5io** [comment]
🚀 Preview for commit ce4ee09f121c4c24150edce3df2738e986b29a6a at: https://675ba2e46168b35f98b78323--layer5.netlify.app

**Comment by mdkaifansari04** [comment]
Looks good @Ajay-singh1, dark and light svg is also working 👍

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
@Ajay-singh1 why thislight mode for the aws only? not other ones like GCP and kubernetes? And why only on the one part and not on the whole page we should use dark and light mode images?

**Comment by Ajay-singh1** [comment]
@sudhanshutech We should use dark and light mode images on the whole part of the page that would be awesome.I didn't find the assets to implement it but now I will go on create those assets and implement this feature.

**Comment by l5io** [comment]
🚀 Preview for commit d10feb70c67803b9cfdbb4634b1d653e2e4192ef at: https://6763b7a7cbfd661d050ecd80--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3dcfeb95261839c4cdf4977bda78dd605b903b51 at: https://6763cba72fc6d46eb810ba71--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ed8522cfd7820da816708c603dda0e5f767373b8 at: https://67641fd866069e221f0ebf33--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ab25f0f63f22a92b530074f399472ea12f0c02d8 at: https://676428b6015266389a663dec--layer5.netlify.app

**Comment by leecalcote** [comment]
Please update the title of this PR to be more descriptive of its purpose and area of focus.

**Comment by Pratik-050** [comment]
Hello @sudhanshutech @leecalcote I was going through this PR when I saw that there are no GIFs for light mode in the repo, we can add light GIFs for better consistency in the light theme.
Screenshot from 2024-12-20 00-14-40

**Comment by Ajay-singh1** [comment]
@Pratik-050 Yes, that would be really great.

**Comment by l5io** [comment]
🚀 Preview for commit f42e9b418f750b97cd4f2a4d0c6c31b847abab43 at: https://6766bd0b44e08b4fb42280b7--layer5.netlify.app

File: src/sections/GCP-Diagram/diagram.js [hunk 1]
```diff
@@ -129,37 +131,11 @@ const DiagramWrapper = styled.div`
 
 const Gcp = () => {
 
-
```
Comment (leecalcote) [inline src/sections/GCP-Diagram/diagram.js:132]: How is this being determined on other pages? Is this method consistent?

Comment (Ajay-singh1) [reply src/sections/GCP-Diagram/diagram.js:132]: Yes! It is , I have made sure the UI looks good in response to different theme.

**Comment by Ajay-singh1** [comment]
@leecalcote If we would have GIF for the light mode it would look really awesome.

**Comment by l5io** [comment]
🚀 Preview for commit d114294bb2065c7b746e6383d3e38539e8d15169 at: https://6766e34a6ec7d85eb180657d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 6e4376d1745f4ad51ef0a487af3625364c6bdfed at: https://676769966d04b0ed9f3a22a5--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@Ajay-singh1 
Let's discuss this during the website call today at 6:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?tab=t.0).

**Comment by l5io** [comment]
🚀 Preview for commit bb6c51a0bcf7d2fbe6426c87daccbfa376c13300 at: https://676a38305a06863a29347c34--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8f89890ba2537ebd4156b0abb5a0be2048f21f81 at: https://676a75568a6ced7e12925ecd--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a876081e23f81d664b049ff0b30bedf0b6529954 at: https://676a7f2a6955888bf13c690c--layer5.netlify.app

**Comment by Satyendra098** [comment]
@Ajay-singh1 3-D Spinning looks good in dark mode.

**Pull request closed without merging**